client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -218,50 +228,132 @@ h2.action-panel__title {
 			}
 		}
 	}
-
-	.email-provider-features {
-		margin-bottom: 2.5em;
-
-		@include breakpoint-deprecated( '>1040px' ) {
-			margin-bottom: 14px;
-			margin-left: 64px;
-			margin-top: 0;
-		}
-	}
 }
 
 .email-provider-stacked-card__provider-right-panel {
-	width: $width-right-panel-card;
+	margin-left: 24px;
 
 	img {
 		margin-top: 40px;
 	}
 }
 
-.action-panel__body {
-	margin-left: 24px;
-}
-
 .email-provider-stacked-card__title {
 	margin-top: 4px;
 	font-size: 1.25rem;
 	line-height: 24px;
 	color: var( --color-neutral-70 );
 }
 
-.email-provider-stacked-card__header {
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	justify-content: space-between;
+.email-provider-stacked-card__title-container {
+	p {
+		margin-bottom: 0;
+		margin-right: 10px;
+	}
+}
+
+.email-providers-stacked-comparison__provider-card {
+	.email-provider-stacked-card__header {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+
+		@include break-small {
+			flex-direction: row;
+		}
+
+		.email-provider-stacked-card__provider-card-main-details {
+			margin-top: 12px;
+			width: 100px;
+			margin-left: auto;
+
+			@include break-small {
+				margin-left: 10px;
+			}
+
+			.email-provider-stacked-card__provider-expand-cta {
+				width: 100%;
+			}
+		}
+	}
+	&.is-expanded {
+
+		.email-provider-stacked-card__header {
+			flex-direction: column;
+			width: 100%;
+
+			@include break-small {
+				flex-direction: row;
+			}
+
+			.email-provider-stacked-card__title-price-badge {
+				width: 100%;
+				@include break-small {
+					width: $width-right-panel-card-expanded;
+				}
+			}
+		}
+	}
 }
 
 .email-provider-stacked-card__provider-card-main-details {
 	width: 120px;
 }
 
 .email-provider-stacked-card__title-container {
-	width: $width-left-panel-card;
-	display:flex;
+	width: 100%;
+	display: flex;
 	flex-direction: column;
+	padding-left: 12px;
+	max-width: 100%;
+
+	@include break-small {
+		width: $width-left-panel-card;
+	}
+}
+
+.email-providers-stacked-comparison__main.main {
+	margin: 12px auto;
+
+	@include break-mobile {
+		flex-direction: row;
+	}
+
+	.card.action-panel.promo-card.email-providers-stacked-comparison__provider-card {
+		display: flex;
+		flex-direction: column;
+
+		.action-panel__figure {
```

Comment:
It looks like there are three `.action-panel__figure` selectors in this file. Can we consolidate them?